pysmt/smtlib/parser/parser.py

```diff
@@ -865,6 +868,69 @@ def get_command_generator(self, script):
             yield cmd
         return
 
+    def parse_model(self, script):
+        """This function pasres the result of a `(get-model)` command and
```

Comment:
This docstring is very clear! :sparkles:

Reply:
:1st_place_medal: